src/event_loop.rs

```diff
@@ -339,14 +339,7 @@ impl<T> EventLoopWindowTarget<T> {
     ///
     /// [`DeviceEvent`]: crate::event::DeviceEvent
     pub fn set_device_event_filter(&self, _filter: DeviceEventFilter) {
-        #[cfg(any(
-            target_os = "linux",
-            target_os = "dragonfly",
-            target_os = "freebsd",
-            target_os = "netbsd",
-            target_os = "openbsd",
-            target_os = "windows"
-        ))]
+        #[cfg(any(free_unix, windows))]
```

Comment:
x11 or windows.

Reply:
clippy will complain about unused functions on Wayland, changed it to x11, wayland or Windows